utils/fo-installdeps

```diff
@@ -12,6 +12,28 @@
 # canonical location for such info, then fix it there first and also
 # update this file and the INSTALL document.
 
+# Line Encoding 
+required_utils=("curl" "dos2unix")
+for util in "${required_utils[@]}"; do
+    if ! command -v $util &> /dev/null; then
+        echo "$util is not installed. Attempting to install..."
+        if [[ $EUID -ne 0 ]]; then
+            echo "Unable to install $util Aborting..."
+            exit 1
+        else
+            case $(uname) in
+                Linux) apt-get install -y $util ;;
+                Darwin) brew install $util ;;
+                *) echo "Unsuported OS." ; exit 1 ;;
+            esac
+        fi
+    fi
+done
+
+if grep -qEi "(Microsoft|WSL)" /proc/version; then
+    find . -type f -print0 | xargs -0 dos2unix
```

Comment:
I will recommend removing from here as well.

Reply:
I've suggested a new approach to writing that can identify files without line encoding and utilizes parallel processing. This is because the only obstacle to cross-platform compatibility between WSL and other Linux distributions is this issue alone.